### ISSUE TITLE
Number of classes 

### ISSUE THREAD

**Issue by ishaanbassi** [mdasari823/VQA-Deep-Learning#5]
In the code, the softmax classifier has 5 classes whereas in the given paper 3 classes have been considered. Can you clarify this ?

**Comment by malleshamdasari** [comment]
In the paper, we wanted only 3 classes. You configure it for however many you would like.

**Comment by ishaanbassi** [comment]
@mdasari823 Actually I am using the pretrained model which predicts 5 classes. How do I interpret the result in this case ? For 3 classes the categories were bad, average and good.

**Comment by malleshamdasari** [comment]
Ah originally, we asked users to rate from 1 to 5 which represents: very bad, bad, average, good, very good. You can interpret this way.
